config/labels/analyzers/clang-tidy.json

```diff
@@ -5085,6 +5092,12 @@
       "profile:sensitive",
       "severity:LOW"
     ],
+    "misc-const-correctness": [
+      "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/misc/const-correctness.html",
+      "profile:extreme",
```

Comment:
I'm not sure whether we should add this to any profile by default, but the severity looks okay.

Reply:
I think it is okay in the extreme profile